### DISCUSSION TITLE
Is it possible to get the list of all the blog post urls?

### DISCUSSION THREAD

**Discussion by x4Cx58x54** [squidfunk/mkdocs-material#6211] [Q&A]
For the non-blog documentation articles in mkdocs, I can get an map containing every one of them with urls using javascipt
```js
var navigation = `{{ navigation }}`;
```
For the blog posts, they are not in the navigation, thus the code above do not get those posts.

I am wondering that if there is a similar interface for the newly landed blog plugin.

**Comment by squidfunk** [comment]
Currently not, but you can explain your use case in https://github.com/squidfunk/mkdocs-material/issues/5074, which is related from a technical perspective.
